client.go

```diff
@@ -198,3 +235,20 @@ func (c Client[Resource]) DeleteById(ctx context.Context, id string, options Opt
 	}
 	return nil
 }
+
+// Delete allow to remove multiple resources.
+func (c Client[Resource]) Delete(ctx context.Context, options Options) error {
```

Comment:
Also this, maybe DeleteMany it's better?

Reply:
yep